fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -24,6 +25,13 @@
 
 #ifdef SSD_ROCKSDB_EXPERIMENTAL
 
+static_assert(ROCKSDB_MAJOR >= 6
```

Comment:
I would just have three separate calls to `static_assert`. I think this logic is correct, but it is very hard to validate by inspection.

Reply:
Done